app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -8,17 +8,14 @@ define([
   'underscore',
   'lib/auth-errors',
   'lib/profile-errors',
+  'models/notifications',
   'models/profile-image'
-], function (_, AuthErrors, ProfileErrors, ProfileImage) {
+], function (_, AuthErrors, ProfileErrors, Notifications, ProfileImage) {
   'use strict';
 
   return {
-    initialize: function (options) {
-      var notifications = this.notifications = options.notifications;
-      if (notifications) {
-        notifications.on(notifications.EVENTS.PROFILE_CHANGE,
-          _.bind(this.onProfileUpdate, this));
-      }
+    notifications: {
+      'profile:change': 'onProfileUpdate'
```

Comment:
Instead of hard coding the notification name, use the format [used here](https://github.com/mozilla/fxa-content-server/pull/3233/files#diff-a378a0bcb0035e7354c7d4662b67c4f0R32)